### ISSUE TITLE
SC tags are not doing any output

### ISSUE THREAD

**Issue by sebabal** [duncanmcclean/simple-commerce#279]
Using the latest version and I am not getting any output from any of the tags eg:

```html
{{ sc:cart:addItem class='flex flex-col' }}
					<input type="hidden" name="product" value="{{ id }}">
					<input type="hidden" name="quantity" value="1">
					
					<div class="flex flex-row items-stretch w-full my-2 px-6">
						{{ if stock === 0 }}
						<p class="text-red-600 font-semibold">Out of stock</p>
						{{ else }}					
						<button x-show="loading === true"
							class="draw-border-tertiary-2 ml-auto py-2 px-6 uppercase text-white font-bold flex flex-row items-center justify-center"
							type="button">
							<i class="fal fa-spinner-third fa-spin mr-2"></i> Adding
						</button>					
						<button x-show="loading === false"
							class="draw-border-white-2 ml-auto py-2 px-6 uppercase text-white font-bold" type="button"
							x-on:click="loading = true; document.forms[0].submit()">
							<i class="fal fa-cart-plus mr-2"></i> Add to Cart
						</button>
						{{ /if }}
					</div>
					{{ /sc:cart:addItem }}
```

## Environment

Statamic 3.0.0-beta.44 Pro
Laravel 6.18.35
PHP 7.4.3

**Comment by duncanmcclean** [comment]
Good catch, it's happening for me too.

**Comment by duncanmcclean** [comment]
Fixed in 1fce44c - missed some stuff when I was upgrading after a refactor of Simple Commerce's tag system. Sorry about that.

**Comment by duncanmcclean** [comment]
Should be good now (https://github.com/doublethreedigital/simple-commerce/releases/tag/v2.0.8)
